flutter/omise/example/android/build.gradle

```diff
@@ -26,6 +26,6 @@ subprojects {
     project.evaluationDependsOn(':app')
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
```

Comment:
https://developer.android.com/build/migrate-to-kotlin-dsl#:~:text=Kotlin%20is%20preferred%20over%20the,time%20checking%20and%20IDE%20support.